oaknational/Oak-Web-Application
# feat/exam-board-search-filter LESQ-479

## Description

Music year: 1968

- Adds exam board search filters
- Points search page at Database-Tools rather than Data-Tools
- Updates SDK
- Updates test files, types and schemas
- Updates the Elastic Query and Test file

## Issue(s)

Fixes #LESQ-479

## How to test

1. Go to https://deploy-preview-2135--oak-web-application.netlify.thenational.academy
2. Click on search page
3. You should see exam board filters
4. Will filter new content by exam board

## Screenshots

How it used to look (delete if n/a):
<img width="365" alt="Screenshot 2023-12-19 at 09 07 22" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/6103b64a-c6db-4458-bc82-3795fb0322ad">

How it should now look:
<img width="428" alt="Screenshot 2023-12-19 at 09 08 03" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/deb259b9-166c-4a01-838c-45d533956812">

## Checklist
